File: pset1/mario.c
#include <stdio.h>
#include <cs50.h>

void pyramid(int n);
int main(void)
{
    int n;
    do
    {
        printf("Height: ");
        n = get_int();
    }
    while (n<0 || n>23);
    pyramid(n);
}

void pyramid(int n)
{
    int hash=1;
    int space=n-1;
    while (n>0)
    {
        for (int i=space;i>0;i--){
            printf(" ");
        }
        for (int i=hash;i>0;i--){
            printf("#");
        }
        printf("  ");
        for (int i=hash;i>0;i--){
            printf("#");
        }
        hash++;
        space--;
        printf("\n");
        n--;
    }
}

File: pset6/mario.py
import cs50

while True:
    print("Height:",end="")
    s=cs50.get_int()
    if s >0 and s<24:
        break

for x in range(s):
    print(" "*(s-x),end="")
    print("#"*(x+1),end="")
    print("  ",end="")
    print("#"*(x+1))